### PR TITLE
Sage developing a smart contract based supply chain management system on celo blockchain using python

### PR DESCRIPTION
**Pull request by Southpaw0**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it, and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the above standard.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/sage-developing-a-smart-contract-based-supply-chain-management-system-on-celo-blockchain-using-python/sage-developing-a-smart-contract-based-supply-chain-management-system-on-celo-blockchain-using-python.md [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+## Next Steps
+
+Your supply chain management system can be improved still more by:
+
+- Creating an intuitive [front-end application](https://docs.celo.org/blog/tutorials/designing-a-user-friendly-celo-dapp-a-beginners-guide-to-uiux) to facilitate interaction with smart contracts.
+- Extending the [system to accommodate a wider range of supply chain actors](https://consensys.net/blockchain-use-cases/supply-chain-management/), including producers, distributors, merchants, and final customers.
+
+## About the Author
+
+[Israel Okunaya](https://meetisraelokunaya.curious.page/) is an ace writer with a flair for simplifying complexities and a knack for storytelling. He leverages over four years of experience to meet the most demanding writing needs in different niches, especially food and travel, blockchain, and marketing. He sees blockchain as a fascinating yet tricky affair. So, he is given to simplifying its complexities with text and video tutorials.
```
Comment (Julius170) [inline blog/sage-developing-a-smart-contract-based-supply-chain-management-system-on-celo-blockchain-using-python/sage-developing-a-smart-contract-based-supply-chain-management-system-on-celo-blockchain-using-python.md:370]: can you add your linkedIn and Github profile link somewhere in this section

**Review by Julius170 — COMMENTED** [review]
Hi @Southpaw0 I've left a commnet on your PR

File: blog/sage-developing-a-smart-contract-based-supply-chain-management-system-on-celo-blockchain-using-python/sage-developing-a-smart-contract-based-supply-chain-management-system-on-celo-blockchain-using-python.md (hunk 1 above)
Comment (Southpaw0) [reply blog/sage-developing-a-smart-contract-based-supply-chain-management-system-on-celo-blockchain-using-python/sage-developing-a-smart-contract-based-supply-chain-management-system-on-celo-blockchain-using-python.md:370]: The links are already contained on my portfolio page.

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

**Pull request merged**